app/enterprise/1.5.x/developer-portal/using-the-editor.md

```diff
@@ -28,49 +28,50 @@ This will launch the **Editor Mode**:
 ![Editor Mode](https://doc-assets.konghq.com/1.3/dev-portal/editor/devportal-editor-mode-launch.png)
 
 
->Note: Editor Mode is a *live* editor - once Saved, changes to files will propagate to the Dev Portal *immediately*. 
+>Note: Editor Mode is a *live* editor. After changes are saved, changes to files will propagate to the Dev Portal *immediately*.
```

Comment:
```suggestion
```
this actually changed right before release of 1.3.0.0 :( 
Changes are not live and will only be saved when save button or cmd/ctrl+s is pressed